### PR TITLE
Fixes error message when no view binding is found.

### PR DESCRIPTION
**Pull request by rjrjr**
And adds explicit visibility to some existing tests to quiet IDE warnings.

### CODE REVIEW

File: workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt [hunk 1]
```diff
@@ -6,9 +6,9 @@ import kotlin.reflect.KClass
 import kotlin.test.assertFailsWith
 
 @OptIn(WorkflowUiExperimentalApi::class)
-class CompositeViewRegistryTest {
+public class CompositeViewRegistryTest {
```
Comment (zach-klippenstein) [inline workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt:9]: We don't need public visibility modifiers in tests. They are explicitly excluded from the gradle plugin, Android Studio is just clueless.

**Review by zach-klippenstein — APPROVED** [review]
Looks good, but the public modifiers for tests is unnecessary.

File: workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt (hunk 1 above)
Comment (rjrjr) [reply workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt:9]: Yeah, but it's also drowning out all the real warnings. Not sure what to do instead.

Comment (rjrjr) [reply workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt:9]: Out of scope for this PR so I'll back them out.

**Review by steve-the-edwards — APPROVED** [review]

**Pull request merged**